src/cps/pc_compat/meson.build

```diff
@@ -0,0 +1,34 @@
+_flex_version = '>= 2.6'
+_bison_version = '>= 2.6'
+prog_flex = find_program('', required : false)
+flex_args = []
+prog_bison = find_program('', required : false)
+if host_machine.system() == 'windows'
+    prog_flex = find_program('win_flex', required : false, version : _flex_vesrion)
+    if prog_flex.found()
+        # This uses <io.h> instead of <unistd.h>
+        flex_args = ['--wincompat']
+    endif
+
+    prog_bison = find_program('win_bison', required : false, version : _bison_version)
+endif
+if not prog_flex.found()
+    prog_flex = find_program('flex', version : _flex_version)
+endif
+if not prog_bison.found()
+    prog_bison = find_program('bison', version : _bison_version)
+endif
+
+pc_scanner = custom_target(
+    'pc_scanner',
+    command : [prog_flex, flex_args, '--outfile=@OUTPUT0@', '--header-file=@OUTPUT1@', '@INPUT@'],
+    input : 'pc.l',
+    output : ['@BASENAME@.scanner.cpp', '@BASENAME@.scanner.hpp'],
```

Comment:
So the CMake docs refer to a "report" of some sort for flex and/or bison. Do you know what that is? Do we care that it's not referred to here?

Reply:
That file, IIUC, is just a dumped version of grammar, and every single state of the generated parser. Something like this:
```
Grammar

    0 $accept: file $end

    1 file: lines
    2     | lines statement

    3 lines: ε
    4      | lines line

    5 line: "\n"
    6     | statement "\n"

    7 statement: property
    8          | assignment

    9 property: name ":" literal_value

   10 assignment: name "=" literal_value

   11 name: "str"
   12     | "blank" "str"
   13     | "str" "blank"
   14     | "blank" "str" "blank"

   15 literal_value: literal
   16              | "blank" literal

   17 literal: ":"
   18        | "str"
   19        | variable
   20        | literal ":"
   21        | literal "str"
   22        | literal variable
   23        | literal "blank"

   24 variable: "$" "{" "str" "}"
```

Reply:
I can see how this would be useful for someone writing more complex parsers and need to decipher why the parser transitioned to state 245 instead of 256, but it probably is not too helpful for a simple parser like this one